src/main/resources/rcs.conf.example

```diff
@@ -3,11 +3,40 @@
 # Membership Service configuration
 # Required
 ms_url=http://localhost
-ms_port=8080
+# Required
```

Comment:
one nit before you merge this: this file has an odd structure. i suggest placing the required confs for the rcs to run (provider_id, ms_url, ms_port, cache_expiration, xmpp) up in the file and the services info at the bottom. also, ms_ip is redundant.

Reply:
Fixed.

Reply:
when testing the integration branch, i noticed the requestService method in CatalogService is double concatenating the pattern "http://" causing a bug. can you please check whether we should remove the "http://" from the rcs.conf values or from the url pattern in CatalogService? it's important to keep the conf file values consistent with the other fogbow services though.

Reply:
You don't need to remove the "http://" prefix from the rcs.conf file values, just change the SERVICE_URL_FORMAT constant into CatalogService class from the branch remore-service-integration.

Reply:
shouldn't this change be done in this pr?

Reply:
No, because the _requestService_ method belongs to the **CatalogService** class, which will be implemented in the **remote-service-integration** branch.

Reply:
sure. thanks for clarifying this.